src/neptune/internal/utils/deprecation.py

```diff
@@ -61,3 +65,17 @@ def inner(*args, **kwargs):
         return inner
 
     return deco
+
+
+def model_registry_deprecation(func):
+    @wraps(func)
+    def inner(*args, **kwargs):
+        warn_once(
+            "Neptune's model registry has been deprecated and will be removed by the end of 2024."
```

Comment:
Let's not be more specific than necessary in this context

```suggestion
            "Neptune's model registry has been deprecated and will be removed in a future release."
```